protocol/x/clob/keeper/liquidations.go

```diff
@@ -222,34 +224,49 @@ func (k Keeper) PlacePerpetualLiquidation(
 
 	// TODO(DEC-1323): Potentially allow liquidating the same perpetual + subaccount
 	// multiple times in a block.
+	perpetualId := liquidationOrder.MustGetLiquidatedPerpetualId()
 	k.MustUpdateSubaccountPerpetualLiquidated(
 		ctx,
 		liquidationOrder.GetSubaccountId(),
-		liquidationOrder.MustGetLiquidatedPerpetualId(),
+		perpetualId,
 	)
 
-	telemetry.IncrCounter(
-		1,
-		metrics.Liquidations,
-		metrics.PlacePerpetualLiquidation,
-		metrics.Count,
-	)
+	labels := []gometrics.Label{
+		metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
+		metrics.GetLabelForBoolValue(metrics.IsBuy, liquidationOrder.IsBuy()),
+	}
 
+	// Stat the number of liquidation orders placed.
 	telemetry.IncrCounterWithLabels(
-		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.BaseQuantums},
-		metrics.GetMetricValueFromBigInt(liquidationOrder.GetBaseQuantums().ToBigInt()),
-		[]gometrics.Label{
-			metrics.GetLabelForIntValue(metrics.PerpetualId, int(liquidationOrder.MustGetLiquidatedPerpetualId())),
-		},
+		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Count},
+		1,
+		append(
+			[]gometrics.Label{
+				metrics.GetLabelForBoolValue(
+					metrics.Unfilled,
+					orderSizeOptimisticallyFilledFromMatchingQuantums == 0,
+				),
+				metrics.GetLabelForBoolValue(
+					metrics.FullyFilled,
+					orderSizeOptimisticallyFilledFromMatchingQuantums == liquidationOrder.GetBaseQuantums(),
+				),
+			},
+			labels...,
+		),
 	)
 
-	telemetry.IncrCounterWithLabels(
-		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Filled, metrics.BaseQuantums},
-		metrics.GetMetricValueFromBigInt(orderSizeOptimisticallyFilledFromMatchingQuantums.ToBigInt()),
-		[]gometrics.Label{
-			metrics.GetLabelForIntValue(metrics.PerpetualId, int(liquidationOrder.MustGetLiquidatedPerpetualId())),
-		},
-	)
+	// Stat the volume of liquidation orders placed.
```

Comment:
Suggestion: might be more useful to aggregrate this value per market and then stat it in `LiquidateSubaccountsAgainstOrderbook`. Up to you.

Reply:
this is also per market (there is a label). 

I think they are effectively the same, and don't think perf is a concern here.

Reply:
Won't this make it difficult to tell how much liquidation volume there was per block?

Reply:
hmm maybe I am missing something - IMO we don't really care about per block info on human readable graphs, right? we usually care about per minute or per 5 minutes, etc. Our alerts are also set up this way. 

If you want exact block info, they should be emitted as logs (similar to MEV logs).

Reply:
Also I don't think you can zoom in to like 1 second precision on datadog. so even if we aggregate / emit as a single metric, we probably won't be able to look at this block in isolation